tensorly/decomposition/_cp.py

```diff
@@ -426,8 +426,6 @@ def parafac(
                 tl.solve(tl.conj(tl.transpose(pseudo_inverse)), tl.transpose(mttkrp))
             )
             factors[mode] = factor
-            if normalize_factors and mode != modes_list[-1]:
```

Comment:
I think the decision to do this only at the end came out of #264, but was in a closed PR. I need to dig it up.